notebooks/Notebook2.md

```diff
@@ -2,33 +2,56 @@
+
+I will address the three critiques above:
+
+I agree. In the end I think that I will need some form of documentation for the language that shows different possibilities for each input group of the language as well as the general structure. I think I will plan to work on that near the end of the project term. Right now I want to focus on other aspects. 
+
+To Prof Ben, I do think I may need some help with my repository. Although It has been working pretty well I ran into a problem this week when trying to implement a small interface for inputting my language. It turns out that codespaces does not know how to build a physical java application type interface without the proper setup in the repository or connections to a remote explorer. I would appreciate help with that sometime next week. 
+
+Sadly I do not think I can simplify the tedious work that I had to do much more. You will see that there were some changes to the files for the player types this week. The changes allowed for simplification of various parts of the player types such as getting rid of the name aspect or adding simpler setters for each type. 
+
 
 ## Description
 
-**TODO:** Fill in this part with information about your work this week:
-important design decisions, changes to previous decisions, open questions,
-exciting milestones, preliminary results, etc. Feel free to include images
-(e.g., a sketch of the design or a screenshot of a running program), links to
-code, and any other resources that you think will help clearly convey your
-design process.
+This week I worked on a few main things such as: building the bride from input to data structure through a parser, Implementing some sort of single line input either via a physical java interface or that being the command line, and looking over and simplifying the player type classes with individual setters per input type vs one big setter that sets all input types. 
+
+I began the week by building a physical java interface for the system that would allow the user to input the language into a text box. This allowed me to easily test inputs and see if it was working properly. This went fairly smoothly until I tried to add a part to the language, this was a simple “start new game log” phrase that was evaluated in the interface portion of the code and created a new game. This would then allow the user to input the regular dsl language. However I kept getting a null pointer error and I could not figure out why because there were no null pointers being passed through. Eventually I figured out that when adding this extra piece of code to create a new game I was building a null game. I ended taking that portion out and it worked great from there on. I may end up going back and retrying something similar later. ([code here](https://github.com/hmc-cs111-spring2023/BBeckerArtifact/blob/main/src/Interface/TextInputInterface.java))
```

Comment:
Thanks for the description of the issue and the link to the changes, I think it's important especially in these reflections to make sure you describe even things you get stuck on as it shows that effort was put in during that time, which is really important for a project like this where time sinks can be detrimental to your progress.